base/Dockerfile

```diff
@@ -579,13 +578,21 @@ RUN \
     bsdtar -xf gitlens-$VS_GITLENS_VERSION.vsix extension && \
     rm gitlens-$VS_GITLENS_VERSION.vsix && \
     mv extension $HOME/.vscode/extensions/eamodio.gitlens-$VS_GITLENS_VERSION && \
-    #Install French Language extension: https://marketplace.visualstudio.com/items?itemName=MS-CEINTL.vscode-language-pack-fr
-    VS_FRENCH_VERSION="1.44.0" && \
-    wget --quiet --no-check-certificate --content-disposition https://marketplace.visualstudio.com/_apis/public/gallery/publishers/MS-CEINTL/vsextensions/vscode-language-pack-fr/$VS_FRENCH_VERSION/vspackage &&\
-    echo "${SHA256fr} MS-CEINTL.vscode-language-pack-fr-${VS_FRENCH_VERSION}.vsix" | sha256sum -c - && \
-    bsdtar -xf MS-CEINTL.vscode-language-pack-fr-$VS_FRENCH_VERSION.vsix extension && \
-    rm MS-CEINTL.vscode-language-pack-fr-$VS_FRENCH_VERSION.vsix && \
+    #Install French Language extension via building and copying https://github.com/microsoft/vscode-loc/tree/release/1.50
```

Comment:
- Why use version 1.50 and not the newer https://github.com/microsoft/vscode-loc/tree/release/1.52 ?
- Start the comment line with space

Reply:
My initial reasoning for using `1.50` as it's the second most up to date and I know that they would not mess with it. I do suppose that if it's under a **release** then they would not change it.

Reply:
Upon further review, I don't think `code-server` is compatible with 1.52 currently. The most recent version I can get to be recognized is `1.50` 
![image](https://user-images.githubusercontent.com/23174198/102933464-4768ab80-4470-11eb-96d7-f35d31065621.png)